dev/io.openliberty.microprofile.graphql.2.0.internal_fat_tck/publish/servers/FATServer/jvm.options

```diff
@@ -0,0 +1,11 @@
+# Uncomment the following to debug remotely
+#-Dwas.debug.mode=true
+#-Dcom.ibm.websphere.ras.inject.at.transform=true
+#-Dsun.reflect.noInflation=true
+#-Dcom.ibm.tools.attach.enable=yes
+#-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=7777
+
+-Xmx2048m
+
+# required for TCK:
+-Dmp.graphql.tck.endpoint.port=8010
```

Comment:
I thought that all FAT port numbers were dynamic? How do you know it will be 8010?

Reply:
That was a carry-over from the previous TCK - it is unnecessary now.  Removed.